probe-rs/src/flashing/flasher.rs

```diff
@@ -194,14 +194,29 @@ impl<'session> Flasher<'session> {
 
     pub(super) fn run_erase_all(&mut self) -> Result<(), FlashError> {
```

Comment:
@Yatekii I do not believe this function is actually called when doing the chip erase cycle?

Reply:
https://github.com/probe-rs/probe-rs/blob/master/probe-rs/src/flashing/loader.rs#L317 it is called in the commit :)

Reply:
Ah I now see the problem. The `started_erasing()` and `finished_erasing()` need to be moved outside the `self.session.has_sequence_erase_all()`. If the chip doesn't have the erase-all sequence, it never enters this condition, and progress never emits the start/stop events